core/src/main/java/com/orientechnologies/orient/core/tx/OTransactionInternal.java

```diff
@@ -1,30 +1,27 @@
-/*
- *
- *  *  Copyright 2010-2016 OrientDB LTD (http://orientdb.com)
- *  *
- *  *  Licensed under the Apache License, Version 2.0 (the "License");
- *  *  you may not use this file except in compliance with the License.
- *  *  You may obtain a copy of the License at
- *  *
- *  *       http://www.apache.org/licenses/LICENSE-2.0
- *  *
- *  *  Unless required by applicable law or agreed to in writing, software
- *  *  distributed under the License is distributed on an "AS IS" BASIS,
- *  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- *  *  See the License for the specific language governing permissions and
- *  *  limitations under the License.
- *  *
- *  * For more information: http://orientdb.com
- *
- */
 package com.orientechnologies.orient.core.tx;
 
-/**
- * Internal class to manage Transactions.
- */
-public class OTransactionInternal {
-  public static void setStatus(final OTransactionAbstract iTx, final OTransaction.TXSTATUS iStatus) {
-    iTx.status = iStatus;
-  }
+import com.orientechnologies.orient.core.db.ODatabaseDocumentInternal;
+import com.orientechnologies.orient.core.db.record.ORecordOperation;
+import com.orientechnologies.orient.core.id.ORID;
+import com.orientechnologies.orient.core.storage.OBasicTransaction;
+
+import java.util.Collection;
+import java.util.Map;
+
+public interface OTransactionInternal extends OBasicTransaction {
```

Comment:
1. Copyright is absent.
2. JavaDoc explaining the meaning of interface is absent. 
3. JavaDocs for methods are absent. Please try to provide meaningful JavaDocs, not formal ones.